content_scripts/WebRTCCoordinator.js

```diff
@@ -14,6 +15,10 @@ createNameSpace('realityEditor.device.cameraVis');
 
             this.ws.addEventListener('open', this.onWsOpen);
             this.ws.addEventListener('message', this.onWsMessage);
+
+            navigator.mediaDevices.getUserMedia({video: false, audio: true}).then((stream) => {
+                this.audioStream = stream;
+            });
```

Comment:
Interesting to see how you added the audioStream, I didn't know you can get it this easily